### PR TITLE
Always consider langpacks and dictionaries as restartless

### LINKED ISSUE #3354: Don't add "restart required" to language packs
Same as #3307; check it with any langpack from:
https://addons.mozilla.org/en-US/firefox/language-tools/

### PR DESCRIPTION
**Pull request by diox**
(If their type is correctly set)

Fix #3354

### CODE REVIEW

File: src/olympia/files/utils.py [hunk 1]
```diff
@@ -151,6 +151,9 @@ class RDFExtractor(object):
     TYPES = {'2': amo.ADDON_EXTENSION, '4': amo.ADDON_THEME,
              '8': amo.ADDON_LPAPP, '64': amo.ADDON_DICT,
              EXPERIMENT_TYPE: amo.ADDON_EXTENSION}
+    # Langpacks and dictionaries, if the type is properly set, are always
+    # considered restartless.
+    ALWAYS_RESTARTLESS_TYPES = ('64', '8')
```
Comment (EnTeQuAk) [inline src/olympia/files/utils.py]: for tired eyes, could we do something like

```python
ALWAYS_RESTARTLESS_TYPES = (TYPES[amo.ADDON_DICT], TYPES[amo.ADDON_LAPP])
```

although it's not much more readable… maybe nevermind :laughing:

**Comment by EnTeQuAk** [comment]
r+wc

File: src/olympia/files/utils.py (hunk 1 above)
Comment (diox) [inline src/olympia/files/utils.py]: Yeah I wanted to remove the numbers hardcoding, and was trying to figure out something clean, but nothing was really satisfying. I'll go with your suggestion, it's not *much* more readable, but I think it's still an improvement.

Comment (diox) [inline src/olympia/files/utils.py]: Ah except I can't, it has not been defined yet... I'd need to move `TYPES` outside the class to do that...

Comment (diox) [inline src/olympia/files/utils.py]: Merging as-is then.

**Pull request merged**